inference-engine/tests/functional/inference_engine/lp_transformations/reshape_transformation.cpp

```diff
@@ -533,6 +533,22 @@ const std::vector<ReshapeTransformationTestValues> testValues = {
             ngraph::element::u8,
             {{ngraph::element::f32}, {}, {{0.1f}, ngraph::element::f32, {1, 1}}}
         }
+    },
+    // U8: no subtract 4D -> 4D: channels are affected
+    {
+        ngraph::Shape({ 1, 64, 320, 1 }),
+        { 0, 2, 3, 1},
+        LayerTransformation::createParamsU8I8(),
+        {
+            ngraph::element::u8,
+            {{ngraph::element::f32}, {}, {{0.1f}, ngraph::element::f32, {}}}
+        },
+        {
+            ngraph::element::u8,
+            {{}, {}, {}},
+            ngraph::element::u8,
+            {{ngraph::element::f32}, {}, {{0.1f}, ngraph::element::f32, {}}}
+        }
```

Comment:
functional test with per-tensor quantization for Subtract + Multiply is skipped

Reply:
Done, thanks!